app/javascript/crayons/MentionAutocomplete/MentionAutocompleteCombobox.jsx

```diff
@@ -71,12 +75,19 @@ export const MentionAutocompleteCombobox = ({
         return;
       }
 
-      fetchSuggestions(trimmedSearchTerm).then((fetchedUsers) => {
+      fetchSuggestions(trimmedSearchTerm).then(({ result: fetchedUsers }) => {
+        const resultLength =
+          fetchedUsers.length > MAX_RESULTS_DISPLAYED
+            ? MAX_RESULTS_DISPLAYED
+            : fetchedUsers.length;
```

Comment:
Does this give us the same result? Just thinking in terms of expressiveness.

```suggestion
        const resultLength = Math.min(fetchedUsers.length, MAX_RESULTS_DISPLAYED);
```

Reply:
nice! yes it's more readable that way I think 👍